cypress/integration/chi-custom-elements/text-input.spec.js

```diff
@@ -0,0 +1,59 @@
+const SELECTORS = {
+  chi_text_input: 'chi-text-input',
```

Comment:
Uppercase here and in every class.

```suggestion
  CHI_TEXT_INPUT: 'chi-text-input',
```